operator/redisfailover/service/generator.go

```diff
@@ -88,6 +88,10 @@ func generateSentinelConfigMap(rf *redisfailoverv1alpha2.RedisFailover, labels m
 	namespace := rf.Namespace
 
 	labels = util.MergeLabels(labels, generateLabels(sentinelRoleName, rf.Name))
+	sentinelConfigFileNameContent := `sentinel monitor mymaster 127.0.0.1 6379 2
```

Comment:
I'd rename this to `sentinelConfigFileContent`

Reply:
I'll change it 😊